index.md

```diff
@@ -9,33 +9,29 @@ headerButtonUrl: "/what-is-scala/"
 
 # Links of the Download / API Docs sections
 gettingStarted:
-  mainTitle: "Scala 3.1.1"
-  mainUrl: "/download/scala3.html"
-  subtitle: "Documentation"
-  subtitleLink: "https://docs.scala-lang.org/scala3"
+  mainTitle: "Get Started"
+  mainUrl: "https://docs.scala-lang.org/getting-started.html"
+  subtitle: "All Scala Releases"
+  subtitleLink: "/download/all.html"
   links:
-    - title: "New in Scala 3"
-      url: "https://docs.scala-lang.org/scala3/new-in-scala3.html"
-    - title: "Scala 3 Book"
-      url: "https://docs.scala-lang.org/scala3/book/introduction.html"
-    - title: "Language Reference"
-      url: "https://docs.scala-lang.org/scala3/reference/overview.html"
-    - title: "Migration Guide"
-      url: "https://docs.scala-lang.org/scala3/guides/migration/compatibility-intro.html"
-apiDocs:
-  mainTitle: "Scala 2.13.8"
-  mainUrl: "/download/scala2.html"
-  subtitle: "Documentation"
-  subtitleLink: "https://docs.scala-lang.org"
-  links:
-    - title: "Current API Docs"
+    - title: "API Documentation"
       url: "https://www.scala-lang.org/api/current/?_ga=1.241039811.1310790544.1468501313"
-    - title: "API Docs (other versions)"
+    - title: "API Docs (Prior Releases)"
       url: "https://scala-lang.org/files/archive/api/"
     - title: "Language Specification"
       url: "https://scala-lang.org/files/archive/spec/2.13/"
-    - title: "All Previous Releases"
-      url: "/download/all.html"
+apiDocs:
+  mainTitle: "Learn Scala"
+  mainUrl: "https://docs.scala-lang.org"
+  subtitle: "Scala Book"
+  subtitleLink: "https://docs.scala-lang.org/scala3/book/introduction.html"
+  links:
+    - title: "New in Scala 3"
+      url: "https://docs.scala-lang.org/scala3/new-in-scala3.html"
+    - title: "Migrate From Scala 2"
+      url: "https://docs.scala-lang.org/scala3/guides/migration/compatibility-intro.html"
```

Comment:
With the focus of the front page shifting away from the Scala 2/3 distinction, I feel like those two items don't really make much sense anymore. Not sure what to replace them with, though.

Reply:
<img width="296" alt="Screenshot 2022-02-24 at 00 26 38" src="https://user-images.githubusercontent.com/13436592/155426717-9c407829-c922-465d-98e7-ca5ef907af03.png">

updated as above